arm/Microsoft.RecoveryServices/vaults/readme.md

```diff
@@ -1,6 +1,6 @@
-# RecoveryServicesVaults `[Microsoft.RecoveryServices/vaults]`
+# Recovery Services Vaults `[Microsoft.RecoveryServices/vaults]`
 
-This module deploys Recovery Service Vault, with resource lock.
+This module deploys a recovery service vault, with resource lock.
```

Comment:
Same comment as before. I'd propose to remove the lock reference if it's not a default deployment

Reply:
done